lib/packages/core/src/__tests__/constructAttachment.spec.ts

```diff
@@ -0,0 +1,46 @@
+import {constructAttachment, Transaction} from '..';
+
+
+describe('constructAttachment', () => {
```

Comment:
TDD - mounted the test before implementing `createParametersFromAttachment`

Reply:
nice one! 🥇